0800-appendix-195943-stock.Rmd

```diff
@@ -175,39 +144,29 @@ fpr::fpr_table_cv_summary_memo(dat = pscis_phase1, site = 1805507, site_photo_id
 habitat was rated as `r fpr::fpr_my_priority_info(loc = 'us')` value as
 an important migration corridor containing suitable spawning habitat and
 having moderate rearing potential.
 
 <br>
 
-Taman Creek was surveyed for
-`r fpr::fpr_my_priority_info(loc = 'us', col_pull = 'length_surveyed')`m
-above where it joins the Bulkley River mainstem upstream of PSCIS
-crossing `r as.character(my_site)`
-`r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-195943-05) - \\@ref(fig:photo-195943-06))")}else(knitr::asis_output("(Figure \\@ref(fig:photo-195943-d03))"))`.
-`r fpr_my_habitat_paragraph(sit = my_site2, loc = 'us')` The water in Taman Creek
-during the survey was noted as turbid with numerous samlonid fry present
-at the downstream end of the site. Algae was abundant throughout the
-system and eroding banks were common. Abundant gravels suitable for
-resident rainbow trout and coho salmon spawning were present and habitat
-was rated as `r fpr::fpr_my_priority_info(sit = my_site2, loc = 'us')` value for salmon
-rearing and spawning.
+`r fpr_my_survey_dist(loc = 'ds', sit = my_site2)`
+`r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-195943-03) - \\@ref(fig:photo-195943-04)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-195943-d02))."))`  
+`r fpr_my_habitat_paragraph(loc = 'us')` There were many undercut banks throughout, with fish up tp 20cm long sighted throughout covered areas. There was a cattle trail next to length of creek, with some areas having obvious signs of embankment issues and an associated risk of sediment falling into stream. There were not many gravel patches suitable for spawning. The
+habitat was rated as `r fpr::fpr_my_priority_info(loc = 'ds', sit = my_site2)` value as
+an important migration corridor containing some suitable spawning habitat and
+having moderate rearing potential.
 
 <br>
 
 ## Stream Characteristics Upstream of `r as.character(my_site2)` {-}
 
+`r fpr_my_survey_dist(loc = 'us', sit = my_site2)`
+`r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-195944-05) - \\@ref(fig:photo-195944-06)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-195944-d02))."))` `r fpr_my_habitat_paragraph(loc = 'us', sit = my_site2)` This section of the stream is likely an important migration corridor, however the stream is heavily impacted by cattle activity. There were significant amounts of cattle manure in the stream, which resulted in water quality levels above thresholds. There were occasional pockets of gravel suitable for resident rainbow spawning. Boulders were the significant cover however there were occasional pools with depths suitable for overwintering. Decent amount of rearing habitat for juveniles though. There was a small rock drop of 65cm located 365m upstream of the top end of the culvert. 
```

Comment:
unless we measured water quality we shouldn't mention that it was above thresholds.  If we mention thresholds we should define them (nah)..